config/gameConfig.json

```
@@ -296,7 +296,12 @@
 			// number of artifacts that can fit in a backpack. -1 is unlimited.
 			"backpackSize"		: -1,
 			// if heroes are invitable in tavern
-			"tavernInvite"            : false
+			"tavernInvite"            : false,
+			// minimai primary skills for heroes
+			"minimalAttack": 0,
+			"minimalDefence": 0,
+			"minimalKnowledge": 1,
+			"minimalSpellPower": 1
```

No need for 4 separate values, especially if they are handled only as array
```suggestion
			"minimalPrimarySkills": [ 0, 0, 1, 1]
```